README.md

```diff
@@ -4,6 +4,14 @@
 
 ## 2018
 
```

Comment:
@sks444 One liner would be great - Something like - `Coala - Shrikrishna Singh - [Newcomer metrics and gamification]`